conda_build/convert.py

```diff
@@ -32,7 +32,7 @@
 @echo off
 set PYFILE=%~f0
 set PYFILE=%PYFILE:~0,-4%-script.py
-"%~f0\\..\\..\\python.exe" "%PYFILE%" %*
+"%~dp0\..\python.exe" "%PYFILE%" %*
```

Comment:
The entry points worked fine before, but the new version is more direct (and maybe more robust). The old one would be 

    C:\path\to\anaconda\Scripts\entrypoint.bat\\..\\..\\python.exe

which works, although somewhat convoluted (and the double slashes are unneccessary). The new one gives

    C:\path\to\anaconda\Scripts\..\python.exe

which is more direct.

Windows is actually very forgiving in evaluating paths with .., as this example shows:

    C:\Users\gom\AppData\Local\Continuum\Miniconda2>this\path\\does\not\\\exist\thisfileneither.exe\..\..\..\..\..\..\python.exe --version
    Python 2.7.12 :: Continuum Analytics, Inc.